libdivide.h

```diff
@@ -229,8 +229,12 @@ static LIBDIVIDE_INLINE struct libdivide_u32_branchfree_t libdivide_u32_branchfr
 static LIBDIVIDE_INLINE struct libdivide_s64_branchfree_t libdivide_s64_branchfree_gen(int64_t d);
 static LIBDIVIDE_INLINE struct libdivide_u64_branchfree_t libdivide_u64_branchfree_gen(uint64_t d);
 
+static LIBDIVIDE_INLINE int16_t libdivide_s16_do(
```

Comment:
These new functions need to have a different name, otherwise this will fail to compile as C, which doesn't have overloading. Maybe something like `libdivide_u16_do_raw` or whatever you think